guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,607 @@
 {
```

Comment:
<p>Can we replace the hyperlink for STAC with this: https://stacspec.org/ (which is more high level and public facing than the previous)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done!